jobs/garden/templates/garden_ctl.erb

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash -e
 
+<% if_p('env.http_proxy') do |http_proxy| %>
+	export HTTP_PROXY=<%= http_proxy %>
+	export http_proxy=<%= http_proxy %>
+<% end %>
+<% if_p('env.https_proxy') do |https_proxy| %>
+	export HTTPS_PROXY=<%= https_proxy %>
+	export https_proxy=<%= https_proxy %>
+<% end %>
+<% if_p('env.no_proxy') do |no_proxy| %>
+	export NO_PROXY=<%= no_proxy %>
+	export no_proxy=<%= no_proxy %>
+<% end %>
```

Comment:
@goonzoid It will better if we keep both cases for the proxy, and if you want to move the block after variable declaration then i can change it else we can finalize the PR.